docs/components/Combobox/Web.stories.tsx

```diff
@@ -197,3 +249,6 @@ ClearSelection.args = {};
 
 export const EmptyState = ComboboxEmptyState.bind({});
 EmptyState.args = {};
+
+export const MultiSelect = ComboboxMultiSelection.bind({});
```

Comment:
is this a good enough name if it's demonstrating both multi select AND onClose?

should I just make another example with `onClose`? 

how many examples is too many?

Reply:
I think that functionality is technically demonstrated here so I don't think another example is necessary. I would lean more towards adding more documentation than adding another example if you think there's something else worth calling out!